### DISCUSSION TITLE
How to automate start script

### DISCUSSION THREAD

**Discussion by marcoideait** [open-wa/wa-automate-nodejs#2793] [Q&A]
Hello I need to automate script that can start command like:
 npx @open-wa/wa-automate -w 'https://myendpoint' 
I've already create a script but process need to be authenticated by qrcode script every time. Ho to fix this, to authenticate one time and then start the script without scan code every time?
Also if I start it by cli with a command like this:
 npx @open-wa/wa-automate -w 'https://myendpoint' & 
The process was killed when I log out from the shell

Is possible to fix qrcode scan and make a service?

Thanks in advance

**Comment by smashah** [comment]
@marcoideait you may want to check this out:

https://github.com/open-wa/wa-automate-nodejs/discussions/2651#discussioncomment-2548732


You can offload the easy api process to pm2 (if installed)
